ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -0,0 +1,425 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: TooltipView
+
+class TooltipView: UIView {
+
+    init(anchorView: UIView,
```

Comment:
it might be worth adding a comment that visually represent the hierarachy how we coded up laying out all the views within the tooltipview. and what offset/size var means what.  ex) https://github.com/microsoft/fluentui-apple/blob/037b7f6254bf83f168cbe97d9a9656f7df58d0e4/ios/FluentUI/SegmentedControl/SegmentedControl.swift#L74